myhpi/static/scss/footer.scss

```diff
@@ -1,5 +1,5 @@
 .footer-myhpi {
```

Comment:
Is it wanted that there is no difference between the webpage background and the content background? The orange bar at the top and the footer at the bottom look a little bit lost to me without the content background

![grafik](https://github.com/user-attachments/assets/518c572d-d5f5-49be-b56f-7a2724ae4b16)

This might also fix the lack of contrast between table cells and content background:

![grafik](https://github.com/user-attachments/assets/8225ff7d-0923-4c90-ad1c-b2603c4f8f2a)